docker/build.mk

```diff
@@ -21,6 +21,7 @@ BASE_TAG ?= gcr.io/fuzzbench
 
 build-all: $(addsuffix -all, $(addprefix build-,$(FUZZERS)))
 pull-all: $(addsuffix -all, $(addprefix pull-,$(FUZZERS)))
+test-run-all: $(addsuffix -all, $(addprefix test-run-,$(FUZZERS)))
```

Comment:
This need a rebase and move to docker/generate_makefile.py

Reply:
We don't have a test-run-all target now but we do have test-run-$FUZZER-all. I think that's probably enough. WDYT?